README.md

```diff
@@ -33,14 +33,25 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the difference between an adaptive website and a fully responsive website?
+   
+* Responsive Web Design provides the optimal viewing experience of a website, no matter what type of device the user is seeing it on where by Adaptive web design is different from responsive design in that there isn’t one layout that always changes. Instead, there are several distinct layouts for multiple screen sizes.
 
-2. Describe what it means to be mobile first vs desktop first.
+1. Describe what it means to be mobile first vs desktop first.
```

Comment:
?

Reply:
Please, could you elaborate on this comment?